build.gradle

```diff
@@ -97,6 +97,15 @@ tasks.register('createReleaseTag', CreateGitTag) {
 
 tasks.named('githubRelease') {
     dependsOn('createReleaseTag')
+    notCompatibleWithConfigurationCache("$name task does not support configuration caching")
+}
+
+tasks.named('publishPlugins') {
+    notCompatibleWithConfigurationCache("$name task does not support configuration caching")
+}
```

Comment:
Will it be CC compatible in version 1.0 of the publish plugin?

Reply:
It does not appear to be cacheable on the current release candidate.